packages/big-design/src/components/List/Item/styled.tsx

```diff
@@ -69,6 +69,19 @@ export const StyledListItem = styled.li<ListItemProps<unknown>>`
   }
 `;
 
+export const StyledLabel = styled.span<any>`
+  margin-right: ${({ theme, isSelected }) => (isSelected ? theme.helpers.remCalc(8) : theme.helpers.remCalc(28))};
```

Comment:
@jorgemoya also I have some doubts about this values. They related to check icon width but now it is not obvious. And if someday icon width will be changed I believe nobody know that this values should be also updated. I will appreciated for some ideas how to improve it.